api/models.py

```diff
@@ -233,3 +233,20 @@ def clean(self):
     def save(self, *args, **kwargs):
         self.full_clean()
         super().save(*args, **kwargs)
+
+
+class UserFeedback(models.Model):
+    """ Stores user feedback data """
+    FEEDBACK = "feedback"
+    BUG = "bug"
+    option = (
+        (FEEDBACK, "feedback"),
+        (BUG, "bug"),
+    )
+    reported_by = models.EmailField(max_length=50, unique=False, null=False)
```

Comment:
the reporter should be an existing user

Reply:
But won't this always save an existing user as the issue of the token comes to play ?

I did try to modify the models to include a the `User` model but I get an extra column in the db and it makes its own user. With this approach I can only post once to the endpoint See the picture from my db
<img width="676" alt="screen shot 2018-04-17 at 14 43 15" src="https://user-images.githubusercontent.com/6042152/38867946-28543038-424e-11e8-9a9d-e1b2ef644548.png">
<img width="726" alt="screen shot 2018-04-17 at 14 43 37" src="https://user-images.githubusercontent.com/6042152/38867948-287e8572-424e-11e8-9733-4be3b6f97b37.png">

Reply:
i think it's ok to have the current user set as the one who's submitted the feedback since to access the app one has to be logged in. Not sure what you mean by the extra column..

Reply:
and by existing user i meant we should go with `ForeignKey` instead of `EmailField `

Reply:
Done